src/symbols.c

```diff
@@ -1404,90 +1404,10 @@ static guint get_tag_class(const TMTag *tag)
 }
 
 
-/* positions a popup at the caret from the ScintillaObject in @p data */
-static void goto_popup_position_func(GtkMenu *menu, gint *x, gint *y, gboolean *push_in, gpointer data)
```

Comment:
Nice that we do not need almost all of this code anymore.

Reply:
Yes.

(But I remember being *really* impressed when Colomban came up with this code after me not knowing how to present the popup, taking all the multimonitor setup into account and invoking the popup using a keyboard.)

Reply:
@techee yet as you know, no amount of cleverness can compete with no code at all :slightly_smiling_face: